github/pulls_reviews.go

```diff
@@ -146,8 +150,12 @@ func (s *PullRequestsService) DeletePendingReview(ctx context.Context, owner, re
 // Read more about it here - https://github.com/google/go-github/issues/540
 //
 // GitHub API docs: https://developer.github.com/v3/pulls/reviews/#get-a-single-reviews-comments
```

Comment:
If you don't mind (because it's outside the scope), can you fix this GitHub API docs link? It should be:

https://developer.github.com/v3/pulls/reviews/#get-comments-for-a-single-review